### PR TITLE
fix for issue #177

### PR DESCRIPTION
**Pull request by tomerk**
fix for issue #177

### CODE REVIEW

**Comment by tomerk** [comment]
@etrain

File: src/main/scala/utils/MatrixUtils.scala [hunk 1]
```diff
@@ -18,9 +18,9 @@ object MatrixUtils extends Serializable {
    * @param mat Input matrix.
    * @return Array of rows.
    */
-  def matrixToRowArray[T](mat: DenseMatrix[T]): Array[DenseVector[T]] = {
+  def matrixToRowArray[T : ClassTag](mat: DenseMatrix[T]): Array[DenseVector[T]] = {
     val matT = mat.t
-    (0 until mat.rows).toArray.map(matT(::, _))
+    (0 until mat.rows).toArray.map(x => DenseVector(matT(::, x).toArray))
```
Comment (shivaram) [inline src/main/scala/utils/MatrixUtils.scala]: Just curious: Does `.copy` or `.toDenseVector` do the same thing ?

Also would be good to add a comment here on why we are doing this

Comment (tomerk) [inline src/main/scala/utils/MatrixUtils.scala]: I'm unsure whether .copy and .toDenseVector do the same thing. But I'll add a comment to make it clear why we're doing this.

**Comment by shivaram** [comment]
LGTM. Thanks @tomerk

**Pull request merged**